src/DataFramesMeta.jl

```diff
@@ -139,6 +141,16 @@ end
 
 fun_to_vec(kw::QuoteNode) = kw
 
+function repair_source(x)
```

Comment:
Is this appropriate? (It helps understanding the code.)
```suggestion
function repair_source(x::AbstractVector)
```

Could also use a more specific name like `to_symbols` or something similar.

Reply:
I added the type annotation. 

However I think the name is good since it doesn't convert everything to `Symbol`s. If given a vector of integers, it leaves it untouched.